Resources/views/CategoryAdmin/tree.html.twig

```diff
@@ -19,20 +19,22 @@ file that was distributed with this source code.
 {% macro navigate_child(collection, admin, root, depth) %}
     <ul{% if root %} class="sonata-tree sonata-tree--toggleable js-treeview"{% endif %}>
         {% for element in collection %}
-            <li class="sonata-ba-list-field" objectId="{{ element.id }}" >
-                <div class="sonata-tree__item"{% if depth < 2 %} data-treeview-toggled{% endif %}>
-                    {% if element.parent or root %}<i class="fa fa-caret-right" data-treeview-toggler></i>{% endif %}
-                    <a class="sonata-tree__item__edit" href="{{ admin.generateObjectUrl('edit', element) }}">{{ element.name }}</a>
-                    <i class="text-muted">{{ element.description }}</i>
-                    {#<a class="label label-default pull-right" href="{{ admin.generateObjectUrl('edit', element) }}">edit <i class="fa fa-magic"></i></a>#}
-                    {% if element.enabled %}<span class="label label-success pull-right"><i class="fa fa-check"></i> {{ admin.trans('active', {}, admin.translationDomain) }}</span>{% endif %}
-                    {% if not element.enabled %}<span class="label label-danger pull-right"><i class="fa fa-times"> {{ admin.trans('disabled', {}, admin.translationDomain) }}</i></span>{% endif %}
-                </div>
+            {% if element %}
```

Comment:
Yes, it is an emptyness check. When element is empty, element.id call crash the view generation.  I am not very fluent with Twig. When I saw this code, I'm asking myself why it entered in the foreach loop, even if element is empty.